plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -60,8 +60,10 @@ public static function get_all() {
 				'title'   => __( 'Reach out to customers', 'woocommerce' ),
 				'plugins' => array(
 					self::get_plugin( 'mailpoet:alt' ),
+					// IMPORTANT: Klaviyo needs to be above Mailchimp as per partnership agreement.
+					// https://ghidorahp2.wordpress.com/2025/01/21/change-requests-for-klaviyo
```

Comment:
```suggestion
					// pdibGW-3XM-p2
```

Reply:
Thanks RJ! Could you give it a final look? 🙏